cmp/dataview/DataView.js

```diff
@@ -57,7 +57,6 @@ class LocalModel {
 
     constructor(model) {
         this.model = model;
-        const {groupRowElementRenderer} = model;
 
         this.addReaction({
             track: () => [model.itemHeight, model.groupRowHeight],
```

Comment:
I think if we're going to support arbitrary components for group renderers, we should support arbitrary heights - otherwise we're relying on the custom component you want to use happening to be the same size as the default. Will implement groupRowHeight on GridModel to bring in line with DVM